src/mame/drivers/seattle.cpp

```diff
@@ -1563,6 +1573,29 @@ void seattle_state::update_widget_irq()
 }
 
 
+READ32_MEMBER(seattle_state::output_r)
+{
+	return m_output;
+}
+
+
+WRITE32_MEMBER(seattle_state::output_w)
+{
+	UINT8 bit;
+	UINT8 op = (data >> 8) & 0x0F;
+	UINT8 arg = data & 0xFF;
+	switch (op) {
+		case 0x07: m_output_mode = arg; break;
+		case 0x0B:
+		switch (m_output_mode) {
+			case 0x04: output().set_value("wheel", arg); break; //wheel motor delta. signed byte.
+			case 0x05: for (bit = 0; bit < 8; bit++) output().set_lamp_value(bit, (arg >> bit) & 0x1); break;
+		}
+		break;
```

Comment:
Can you clean this up a little?  You can `const`-qualify `op` and `arg`, case labels don't introduce a level of scope, perhaps you should put default cases that log unknown ops or something, the case label, `for` loop and break on a single line is borderline unreadable, `bit` is not used before or after the loop so it should be declared in the loop not at the top of the function, the rest of this file uses Allman style, not K&R.